eclipse/openvsx
# Do a manual (non-cli) upload

Added publish button to header
Added extensions tab in user settings
Added ability to publish extension
Added ability to create namespace
Make namespaces visible for contributors

Testing steps:
A publish button has been added to the navigation menu.
Click the publish button, the rest of the UX should speak for itself.
